src/Kerbero.Common/Exceptions/CouldNotSaveNukiAccount.cs

```diff
@@ -0,0 +1,13 @@
+using Kerbero.Common.Entities;
+
+namespace Kerbero.Common.Exceptions;
+
+public class CouldNotSaveNukiAccount: Exception
```

Comment:
non capisco bene le motivazioni per questa eccezzione

Reply:
If no rows was written in db, it means that no NukiAccount was been uploaded